packages/sitecore-jss-react/src/index.ts

```diff
@@ -103,4 +103,9 @@ export { withDatasourceCheck } from './enhancers/withDatasourceCheck';
 export { EditFrameProps, EditFrame } from './components/EditFrame';
 export { ComponentBuilder, ComponentBuilderConfig } from './ComponentBuilder';
 export { withFieldMetadata } from './enhancers/withFieldMetadata';
+export { withEmptyValueEditingPlaceholder } from './enhancers/withEmptyValueEditingPlaceholder';
```

Comment:
if the HOC logic will not be merged into withFieldMetadata, please re-export it from jss-nextjs package too for JSS developers to use